satellite/satellitedb/migrate.go

```diff
@@ -905,6 +905,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`UPDATE nodes SET disqualified=NULL WHERE disqualified IS NOT NULL AND audit_reputation_alpha / (audit_reputation_alpha + audit_reputation_beta) >= 0.6;`,
 				},
 			},
+			{
```

Comment:
we might need to change the initial create query, because that would still fail during migrations.

Reply:
oh yeah good point.

Reply:
_updated_